c3web-server/src/main/scala/com/ifunsoftware/c3web/annotation/Vocabulary/ControlledVocabulary/ControlledVocabulary.scala

```diff
@@ -0,0 +1,50 @@
+package com.ifunsoftware.c3web.annotation.vocabulary.controlledVocabulary
+
+import com.ifunsoftware.c3web.annotation.KeyWord
+import com.ifunsoftware.c3web.annotation.vocabulary.Vocabulary
+import com.ifunsoftware.c3web.models.Tag
+import shapeless.LT.<
+
+/**
+ * Created by admin on 17.02.2016.
+ */
+class ControlledVocabulary(content: String) extends Vocabulary {
+  val terms = content.split('\n').distinct;
+
+  def GetAll(): List[KeyWord] = {
```

Comment:
Let's use Scala naming conventions ( [http://docs.scala-lang.org/style/naming-conventions.html](http://docs.scala-lang.org/style/naming-conventions.html) ): methods are camel-cased and start with a lowercase letter.